teamworkApp/lib/evaluateAnswers.py

```diff
@@ -0,0 +1,49 @@
+from enum import IntEnum
+import sqlite3
+
+DB = 'db/development.sqlite3'
+
+Style = IntEnum('Style', 'Contributor, Collaborator, Communicator, Challenger', start=0)
+Answer_Value = IntEnum(
+    'Answer_Value', 
+    '1234 1243 1324 1342 1423 1432 2134 2143 2314 2341 2413 2431 3124 3142 3214 3241 3412 3421 4123 4132 4213 4231 4312 4321',
+    start=0
+)
+
+# the order of answers repeat every 4 questions so just the index is (question_num % 4) 
+Questions = {
+    0: (Style.Contributor, Style.Collaborator, Style.Communicator, Style.Challenger),
+    1: (Style.Collaborator, Style.Communicator, Style.Challenger, Style.Contributor),
+    2: (Style.Communicator, Style.Challenger, Style.Contributor, Style.Collaborator),
+    3: (Style.Challenger, Style.Contributor, Style.Collaborator, Style.Communicator),
+}
+
+def get_students_answers(student_id):
+    """return list of complete answers for a given student"""
+    conn = sqlite3.connect(DB)
+    c = conn.cursor()
+
+    c.execute(
+        'SELECT * FROM answers WHERE student_id=?', 
+        [student_id,],
+    )
+    rows = c.fetchall()
+
+    conn.commit()
+    conn.close()
+
+    return rows
+
+def find_scores(student_id):
```

Comment:
I find myself wanting more comments on this function and a more in-depth docstring.